core/services.go

```diff
@@ -114,14 +130,15 @@ func (c *Core) SelectAllServices(start bool) ([]*Service, error) {
 		for _, f := range fails {
 			service.Failures = append(service.Failures, f)
 		}
-		checkins := service.AllCheckins()
+		checkins := service.AllCheckins() // whaaaatttt?
```

Comment:
`AllCheckins` will check for Checkin's. Checkin's are a scheduled requests to Statping from a service, rather than Statping checking for the service to be online.

Reply:
:) I forgot to delete this. But, wouldn't that be huge after a while?